### PR TITLE
Fix the typo in class name Symfony, not Symphony

### PR DESCRIPTION
**Pull request by harikt**
Added class_alias in config/dms.php .

I think that should work. Let me know if there is any other issue.

### CODE REVIEW

**Comment by harikt** [comment]
Fixes https://github.com/dms-org/web.laravel/issues/2

**Comment by harikt** [comment]
The tests looks already failing, not specifically due to this change.

File: config/dms.php [hunk 1]
```diff
@@ -1,4 +1,6 @@
 <?php
+// https://github.com/dms-org/web.laravel/issues/2
+class_alias(Dms\Web\Laravel\Action\InputTransformer\SymfonyToDmsUploadedFileTransformer::class, Dms\Web\Laravel\Action\InputTransformer\SymphonyToDmsUploadedFileTransformer::class, Dms\Web\Laravel\Action\InputTransformer\SymfonyToDmsUploadedFileTransformer::class, Dms\Web\Laravel\Action\InputTransformer\SymphonyToDmsUploadedFileTransformer::class, true);
```
Comment (harikt) [inline config/dms.php]: I think this was not what you were looking at.

I have a very limited experience with laravel as a framework. Browsed a bit on the package documentation and seems this config is published to app.

So in that case we may need to keep a class that extends `SymfonyToDmsUploadedFileTransformer` may work. Oh and looks like I made a mistake copying and pasting the class name to keep alias :P .

**Comment by TimeToogo** [comment]
Thanks for your contribution.

The tests seem to be failing due to an incompatibility of the testing library with a newer version of laravel which is unrelated to this update. Hopefully that gets resolved shortly.

**Pull request merged**